packages/preact-query-persist-client/src/__tests__/PersistQueryClientProvider.test.tsx

```
@@ -11,7 +11,7 @@ import { notifyManager } from '../../../query-core/src'
 import { act, cleanup, render } from '@testing-library/preact'
 import type { UseQueryResult } from '../../../preact-query/src'
 import { QueryClient, useQuery } from '../../../preact-query/src'
-import { afterEach, beforeEach, describe, expect, test, vi } from 'vitest'
+import { afterEach, beforeEach, describe, expect, vi, it } from 'vitest'
```

ESLint

[error] Member 'it' of the import declaration should be sorted alphabetically.

(sort-imports)